apps/linebot/app.py

```diff
@@ -60,67 +66,36 @@ def reply(intent, text, reply_token, id, disname):
             text_message = TextSendMessage(
                 text="กรุณาใส่รายละเอียดให้ถูกต้องอีกครั้ง")
         line_bot_api.reply_message(reply_token, text_message)
+
+    '''remove subscriber from database'''
     if intent == 'remove_subscribe':
         # remove user's subscription
         print("Removing user from database", flush=True)
         remove_subscriber(id)
         text_message = TextSendMessage(
             text="ดำเนินการลบ subscription ของคุณเรียบร้อย")
         line_bot_api.reply_message(reply_token, text_message)
+
     if intent == 'test':
         text_message = TextSendMessage(text="test success")
         line_bot_api.reply_message(reply_token, text_message)
+
     if intent == 'profile':
         user_profile = update_subscriber(id)
         if len(user_profile) == 0:
             text_message = TextSendMessage(
                 text="เรายังไม่พบข้อมูลของคุณ%s กรุณาลงทะเบียนก่อน" % disname)
         else:
-            text_message = TextSendMessage(text="ข้อมูลส่วนบุคคลของคุณ %s คือ\nกลุ่มโลหิต : %s\nเขต : %s"
-                                           % (disname, user_profile[0].bloodtype, user_profile[0].district))
+            text_message = TextSendMessage(text="ข้อมูลส่วนบุคคลของคุณ %s คือ\nกลุ่มโลหิต : %s\nเขต : %s" % (
+                disname, user_profile[0].bloodtype, user_profile[0].district))
         line_bot_api.reply_message(reply_token, text_message)
 
 
-@app.route('/hello/<name>')
```

Comment:
This no other piece code that multicast the Line Bot message, removing this may be detrimental to our progress. Let's keep this as a placeholder code for future.